platform/feature-api/api/rdf-convert/src/index.ts

```diff
@@ -45,7 +47,7 @@ export function convert(input: RDF.Term, dataFactory: RDF.DataFactory<any>): RDF
         case "Variable":
             if (dataFactory.variable) return dataFactory.variable(term.value);
             else throw new Error("Variables are not supported");
-        default:
+        case "Quad":
```

Comment:
I suppose to maintain semantics, but still have an explicit case for `Quad`, we could do:

```
case "Quad":
default:
    ...
```